README.rst

```diff
@@ -37,21 +39,51 @@ This code runs on Python 3.5 or newer.
 Docker
 ------
 
-Alternatively, you can build and run the xblock-sdk in Docker.
+Alternatively, you can build and run the xblock-sdk in Docker (we are using docker-compose which
+can be installed as explained at https://docs.docker.com/compose/install/)
 
 After cloning this repository locally, go into the repository directory and build the Docker image::
 
-    $ docker build -t xblock-sdk .
+    $ make docker_build
+
+or manually run
+
+    $ docker-compose build
 
 You can then run the locally-built version using the following command::
 
-    $ docker run -d -p 8000:8000 --name xblock-sdk xblock-sdk
+    $ make dev.up
+
+or manually run
+
+    $ docker-compose up -d
+
+and shut-down the container by::
+
+    $ make dev.down
+
+or manually run
+
+    $ docker-compose down
+
+On the first startup run the following command to create the SQLite database.
+(Otherwise you will get an error no such table: workbench_xblockstate.)
+
+Command::
+
+    $ docker container exec -it edx.devstack.xblock-sdk python3.8 manage.py migrate
 
 You should now be able to access the XBlock SDK environment in your browser at http://localhost:8000
 
+You can open a bash shell in the running container by using::
+
+    $ docker container exec -it edx.devstack.xblock-sdk bash
```

Comment:
Can you also add a make command `dev.shell` for this?

Reply:
I added it as `app.shell` to match the sample in https://github.com/edx/edx-cookiecutters/blob/master/cookiecutter-django-ida/%7B%7Bcookiecutter.repo_name%7D%7D/Makefile